Sources/SwiftCurrent_SwiftUI/Views/ViewControllerWrapper.swift

```diff
@@ -20,15 +20,20 @@ public struct ViewControllerWrapper<F: FlowRepresentable & UIViewController>: Vi
 
     public weak var _workflowPointer: AnyFlowRepresentable?
 
+    private let vc: F
     private let args: WorkflowInput?
     public init(with args: F.WorkflowInput) {
         self.args = args
+        vc = F._factory(F.self, with: args)
```

Comment:
It's highly unfortunate that the view control has to be instantiated here AND in the makeUIViewController function. Unfortunately I do not see a way around that, but I wanted to explicitly call it out so nobody was surprisedor glazed over what happened.